roles/etcd_common/defaults/main.yml

```diff
@@ -1,6 +1,7 @@
 ---
 # etcd server vars
-etcd_conf_dir: /etc/etcd
+etcd_conf_dir: "{{ '/etc/etcd' if not openshift.common.is_etcd_system_container else '/var/lib/etcd/etcd.etcd/etc'  }}"
+etcd_system_container_conf_dir: /var/lib/etcd/etc
```

Comment:
unfortunately the rhel/etcd system container reads the `/etc` from the container itself.  I thought of using this for now and then change the etcd system container to accept optionally a bind mount from the host `/etc`